### PR TITLE
Readded pane component

### PR DESCRIPTION
**Pull request by anneback**

### CODE REVIEW

File: src/components/pane/pane.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  }
+
+  handleTabClick(index) {
+    this.setState({
+      activeTab: index,
+    });
+  }
+
+  renderTabs() {
+    const tabWidth = 100 / this.props.tabs.length;
```
Comment (iamstarkov) [inline src/components/pane/pane.jsx]: nope

File: src/components/pane/pane-styles.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      borderBottom: `1px solid ${palette.background.secondary}`,
+    },
+
+    tab: {
+      display: 'inline-block',
+      background: palette.background.default,
+      lineHeight: 1.2,
+      textAlign: 'center',
+      cursor: 'pointer',
+      letterSpacing: 1,
```
Comment (iamstarkov) [inline src/components/pane/pane-styles.jsx]:
```js
width: props => `${100 / props.tabs.length}%`
```

**Comment by iamstarkov** [comment]
Sorry, you cant do it now =((

File: src/components/pane/pane.jsx (hunk 1 above)
Comment (bstream) [inline src/components/pane/pane.jsx]: Rather do this with CSS, see this fiddle: https://jsfiddle.net/j83xnwst/

**Pull request merged**